docs/guides/cdk/fragments/js/importing-configuration.md

```diff
@@ -0,0 +1,12 @@
+### Importing the configuration files
```

Comment:
(question) why is this a separate fragment?

Reply:
I did this separately because it only shows the JS configuration.